### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Modules_Screen Reader: In Capslock+M mode, Narrator is reading “Edit button” and “Delete button” as just button.

### ISSUE THREAD

**Issue by v-supupp** [Microsoft/IIS.WebManager#85]
**Environment Details:**
URL: https://manage.iis.net/webserver/modules
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator, NVDA
**WCAG ID Violated:** WCAG1.3.1

GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #C+E-H; #.NETWebsites; #Edge; #Chrome; #Narrator; #NVDA; #WCAG1.3.1

**Repro Steps:**
1.	Download and install "Microsoft IIS Administration" application.
2.	Enable Internet information Services.
3.	Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4.	Tab till Web Server and press enter
5.	Tab till Modules and press enter
6.	Start Screen Reader (Narrator/NVDA).
7.	Validate that screen reader is reading the name of controls.

**Actual Result:**
In Capslock+M mode, Narrator is reading “Edit button” and “Delete button” as just button. It is not reading their name along with role.

**Expected Result:**
In Capslock+M mode, Narrator should read those controls as “Edit Button” and “Delete Button”. It should read name along with role.

**Note:**

This issue is also observed in the following page:
Webserver->Request Tracing

**User Impact:**
Users with visual disabilities could not understand the behaviour of the controls if screen reader is not reading their names.

**Attaching video for your reference:**
[Bug 85_Narrator_Edit and Delete Button.pptx](https://github.com/Microsoft/IIS.WebManager/files/2318264/Bug.85_Narrator_Edit.and.Delete.Button.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by fsteffi** [comment]
#A11ySev2;

**Comment by Yashi1201** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
